src/dls_bluesky_core/core/types.py

```diff
@@ -1,8 +1,11 @@
-from typing import Any, Callable, Generator
+from typing import Any, Callable, Generator, Union
 
 from bluesky import Msg
+from ophyd_async.core import Device
+from scanspec.specs import DURATION
 
 #  'A true "plan", usually the output of a generator function'
 MsgGenerator = Generator[Msg, Any, None]
 #  'A function that generates a plan'
 PlanGenerator = Callable[..., MsgGenerator]
+ScanAxis = Union[str, Device, DURATION]
```

Comment:
I wanted something that could be operated on by a flyer, which would have all string keys filled with a valid Device (apart from DURATION)

```suggestion
ScannableAxis = Union[Device, DURATION]
```

Reply:
Blame @callumforrester for the change, I'll revert ;)